diff --git a/node_modules/@rbxts/profileservice/src/init.lua b/node_modules/@rbxts/profileservice/src/init.lua
index 02f0378..3500c41 100644
--- a/node_modules/@rbxts/profileservice/src/init.lua
+++ b/node_modules/@rbxts/profileservice/src/init.lua
@@ -2302,9 +2302,9 @@ if IsStudio == true then
 
 			UseMockDataStore = true
 			ProfileService._use_mock_data_store = true
-			print("[ProfileService]: Roblox API services unavailable - data will not be saved")
+			-- print("[ProfileService]: Roblox API services unavailable - data will not be saved")
 		else
-			print("[ProfileService]: Roblox API services available - data will be saved")
+			-- print("[ProfileService]: Roblox API services available - data will be saved")
 		end
 		IsLiveCheckActive = false
 	end)
